build/Dockerfile.libbpf.base

```diff
@@ -12,9 +12,15 @@ RUN yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.n
 # install less frequently updated pkg first
 RUN yum install -y kmod xz python3 && yum clean all -y
 
+# install BCC tools and dependencies
```

Comment:
Only needs `perf` and `bpftrace` for debugging purpose. 

```bash
yum install -y perf bpftrace
```

Reply:
@rootfs now please review the changes.